src/Sarif.Viewer.VisualStudio/Views/CodeLocations.xaml

```diff
@@ -153,6 +154,23 @@
                 <ContentControl ContentTemplate="{StaticResource AnnotatedCodeLocationCollectionDataGridTemplate}"
                                 Content="{Binding CodeFlows}" />
             </TabItem>
+            <TabItem Visibility="{Binding CallTrees, Converter={StaticResource collectionToVisibility0Converter}}">
+                <TabItem.Header>
+                    <DockPanel Margin="5 0 5 0">
+                        <TextBlock DockPanel.Dock="Left"
+                                   Text="Call Trees" />
+                        <TextBlock DockPanel.Dock="Left"
+                                   Visibility="{Binding CallTrees, Converter={StaticResource collectionToVisibility1Converter}}">
+                            <Run Text=" (" />
+                            <Run Text="{Binding CallTrees, Mode=OneWay, Converter={StaticResource collectionToCountConverter}}" />
+                            <Run Text=")" />
+                        </TextBlock>
+                    </DockPanel>
+                </TabItem.Header>
+
+                <ContentControl ContentTemplate="{StaticResource CallTreeTemplate}"
```

Comment:
Here's where we display the actual call trees, using the "template" defined in CallTrees.xaml, above.